README.md

```diff
@@ -1,8 +1,26 @@
-# mealpy 
+# mealpy
+
 Reserve your meals on MealPal automatically, as soon as the kitchen opens.
 Never miss your favourite MealPal meals again!
 
+## Help
+
+```bash
+python mealpy.py --help
+```
+
+## Reserve a meal
+
+```bash
+# python mealpy.py reserve RESTAURANT RESERVATION_TIME CITY
+python mealpy.py reserve "Coast Poke Counter - Battery St." "12:15pm-12:30pm" "San Francisco"
```

Comment:
Minor nit, use invalid data in doucmentation. It's very likely someone will copy-paste this, even though it makes no sense. [Good times](https://np.reddit.com/r/cscareerquestions/comments/6ez8ag/accidentally_destroyed_production_database_on/)

I would suggest just appending something obviously fake onto the full name, so the user still has a good sense of what is expected (at least until we make the valid arguments more discoverable). i.e. "Not San Francisco"

Reply:
Isn't it better to use valid data in examples like these? If a user wants to try out the functionality, they can easily copy-paste the example and see what happens. Then, if they want to customize the restaurant to whatever they want, they have a reference point of what worked.

Reply:
I would be more okay with real example if we had a dry-run mode. Copy-pasting and running this will have real-world side effects, that they may not want, if they're simply trying it out.

This best practice came from talk on incident response runbooks. There was an incident and the runbook had example command with real host. The devs ran the command (under stress) without replacing the host names. Now they have yet another host out of commission. Just an anecdotal example explaining the rationale behind something as innocuous as this.

At the end of the day, it'll be their fault, since these are the literal instructions for how to actually run the script, not simply preview. So I don't mind if you drop this.

Reply:
I think it should be fine in this case.